### ISSUE TITLE
set value to 'false' in 'globals' is invalid ? 

### ISSUE THREAD

**Issue by zoutaojlq** [eslint/eslint#6134]
ESLint version : 2.9
ESLint configuration :
```js
{
  "extends": "eslint:recommended",
  "env": {
    "amd": true,
    "commonjs": true
  },
  "parserOptions": {
    "sourceType": "module"
  },
  "globals": {
    "ui": true,
    "page": false
  }
}
```

source code:
```js
ui = "456"; 
page = "123";
```

actual ESLint output :
```js
ui = "456"; //correct
page = "123"; //correct，There is no report errors，why?
```

I want to make the variable "page" read-only，but invalid.

**Comment by eslintbot** [comment]
Hi @zoutaojlq, thanks for the issue. It looks like there's not enough information for us to know how to help you. If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

If it's something else, please just provide as much additional information as possible. Thanks!

**Comment by zoutaojlq** [comment]
ESLint version : 2.9
ESLint configuration :
{
"extends": "eslint:recommended",
"env": {
"amd": true,
"commonjs": true
},
"parserOptions": {
"sourceType": "module"
},
"globals": {
"ui": true,
"page": false
}
}

source code:
ui = "456"; 
page = "123"; 

actual ESLint output :
ui = "456"; //correct
page = "123"; //correct，There is no report errors，why?

I want to make the variable "page" read-only，but invalid.

**Comment by zoutaojlq** [comment]
?

**Comment by ilyavolodin** [comment]
Sorry, the message was lacking formatting and I didn't notice `extends` in the config. This is a duplicate of #5358 we accidentally lost ability to track changes to read-only global variables at some point. Right now, there is no rule to check for that.
Closing as duplicate of #5358
